tests/test_route_def.py

```diff
@@ -256,6 +272,17 @@ def test_repr_route_def_with_extra_info():
     assert repr(rd) == "<RouteDef GET /path -> 'handler', extra='info'>"
 
 
+def test_repr_static_def():
+    routes = web.RouteTableDef()
+
+    routes.static('/prefix', '/path', name='name')
```

Comment:
`@`

Reply:
No, this method is intentionally not a decorator: static has no user-supported handler, only `prefix`, `path` and optional parameters. I'll document it properly.

The reason why `url.static()` and `routes.static` should exist is it is a part of configuration which very often is constant (safe to be done at module level). No need to add `app.add_statiic()` shortcut.

`app.router.add_static()` is redundant (but still exist and do all work internally).
`app.add_subapp` is a different beast: it requires `subapp` instance, that's why I don't want to add shourcuts for `add_subapp`.

Reply:
Aha, I see. Now it looks fine. On previous version there was a function right after and that looks like missed @ from decorator.